client/src/pages/CreateContest/CreateContestForm/CreateContestForm.tsx

```diff
@@ -1,144 +1,228 @@
+                    </Grid>
+                  </MuiPickersUtilsProvider>
+                  <Grid item xs={2}>
+                    <TextField
+                    id="zone"
+                    name="zone"
                     margin="normal"
-                    variant="inline"
-                    inputVariant="outlined"
-                    value={date}
-                    onChange={newDate => setDate(newDate)}
-                    keyboardIcon={<AlarmIcon />}/>
+                    variant="outlined"
+                    select
+                    value={zone}
+                    onChange={value => setZone(value.target.value)}>
+                      <MenuItem value="PDT">PDT</MenuItem>
+                      <MenuItem value="CDT">CDT</MenuItem>
+                      <MenuItem value="EDT">EDT</MenuItem>
+                    </TextField>
                   </Grid>
-                </MuiPickersUtilsProvider>
-                <Grid item xs={2}>
-                  <TextField
-                  id="zone"
-                  margin="normal"
-                  variant="outlined"
-                  select
-                  value={zone}
-                  onChange={newZone => setZone(newZone.target.value)}>
-                    <MenuItem value="PDT">PDT</MenuItem>
-                    <MenuItem value="CDT">CDT</MenuItem>
-                    <MenuItem value="EDT">EDT</MenuItem>
-                  </TextField>
                 </Grid>
               </Grid>
             </Grid>
-          </Grid>
-        </Box>
-        <Box mt={3} mb={3} className={classes.box}>
-          <Typography className={classes.label}>Which designs do you like?</Typography>
-          <Box mt={3} mb={3} className={classes.imageList}>
-            <ImageList rowHeight={160} cols={4} className={classes.images}>
-              {images.map((image: string) => (
-                <ImageListItem key={image}>
-                  <img src={image} className={classes.img}/>
-                </ImageListItem>
-              ))}
-            </ImageList>
           </Box>
-        </Box>
-        <Box mt={3} mb={3}>
-          <Button
-          type="submit"
-          size="large"
-          variant="contained"
-          color="primary"
-          className={classes.submit}>
-            Create Contest
-          </Button>
-        </Box>
-      </form>
-    </Box>
+          <Box mt={3} mb={3} className={classes.box}>
+            <Typography className={classes.label}>Which designs do you like?</Typography>
+            <Typography className={classes.sub} component='p'>
+              {'Let\'s start by helping your designers understand which styles your prefer.'}
+            </Typography>
+            <Box mt={3} mb={3} className={classes.imageList}>
+              <ImageList rowHeight={160} cols={4} className={classes.images}>
+                {dogImages.map((image: string, id: number) => (
+                  <ImageListItem key={id}>
```

Comment:
Avoid using indexes as keys. Can we use something unique here?

Ref: https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318